### ISSUE TITLE
ListView control does not support UIA accessibility

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3223]
* .NET Core Version: 

.NET Core 5.0
Version: 5.0.100-alpha.1.20073.10
Commit: 29f4d693a9
Runtime Environment:
OS Name: Windows
OS Version: 10.0.18363
OS Platform: Windows
RID: win10-x64
Base Path: C:\Program Files\dotnet\sdk\5.0.100-alpha1-05536
Host (useful for support):
Version: 5.0.0-alpha.1.20072.3
Commit: c3dc1fdfdc

* Have you experienced this same bug with .NET Framework?: 

Yes

**Problem description:**

**Providers:**
[pid:28344,providerId:0x31378 Main:Nested [pid:26684,providerId:0x31378 Main(parent link):Microsoft: ListView Proxy (unmanaged:UIAutomationCore.dll)];
Nonclient:Microsoft: Non-Client Proxy (unmanaged:uiautomationcore.dll);
Hwnd(parent link):Microsoft: HWND Proxy (unmanaged:uiautomationcore.dll)]"

![image](https://user-images.githubusercontent.com/23213980/81116178-94256380-8f2d-11ea-8b59-c2c25d9291a2.png)

**Expected behavior:**
There should be UIA accessibility provider from WinForms

**Minimal repro:**
Create WinForms app with ListView control and analyze it using Inspect.

**Related item:**
This work item is the sub of #2164
